quickstarts/synthetics/free_tier/config.yml

```diff
@@ -0,0 +1,32 @@
+id: 260027d0-9d8f-4a35-a676-73b5ee6c0d2d
+# Name of the quickstart (required)
+slug: free-tier
+
+# Displayed in the UI (required)
+title: Free Teir
+
+# Long-form description of the quickstart (required)
+description: |
+  # We find Free tier
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+  Free tier for services
+
+# Support level: New Relic | Verified | Community (required)
+level: New Relic
+
+# Authors of the quickstart (required)
+authors:
+  - New Relic
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - free
```

Comment:
Are there any other relevant keywords? Take a look at some other similar quickstarts to see what keywords they use.